docs/commands/push.md

```diff
@@ -1,93 +1,186 @@
+- [Travis CI documentation](https://docs.travis-ci.com/user/environment-variables/)
+- [CircleCI documentation](https://circleci.com/docs/2.0/env-vars/)
+- [GitHub Actions documentation](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets)
+- [Jenkins documentation](https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#handling-credentials)
 
+## Usage
 
 ```bash
+openapi push <entrypoint> <destination> [branchName]
+openapi push [--upsert] <entrypoint> <destination> [branchName]
+openapi push 
 openapi push [-u] [--run-id id] <path/to/definition.yaml> <@organization-id/api-name@api-version> [branchName]
 ```
 
+## Options
+
+option           | type      | required?    | default     | description
+-----------------|:---------:|:------------:|:-----------:|------------
+`entrypoint`     | `string`  | yes          | -           | Path to the API definition filename that you want to push to the Redocly API registry. (See [the section below](#entrypoints) for more options)
+`destination`    | `string`  | yes          | -           | Organization id, API name and API version that define the location in the Redocly API registry where to push your API definition.<br />**Format:** `@organization-id/api-name@api-version`. (See [the section below](#destination) for more information)
+`branchName`     | `string`  | no           | `main`      | Branch name that will act as a default one for pushing API definitions to
+`--help`         | `boolean` | no           | -           | Show help
+`--run-id`       | `string`  | no           | -           | CI job ID that the current push will be associated with. See [the section below](#run-id) for more information
+`--upsert`, `-u` | `boolean` | no           | -           | Create a new API and a new API definition version
+`--version`      | `boolean` | no           | -           | Show version number
+
+## Examples
+
+### Entrypoint
 
-Example output:
+The command behaves differently depending on how you pass an entrypoint to it and whether the [configuration file](#custom-configuration-file) exists. There are the following options:
 
-```shell
-Bundling definition
-Created a bundle for test.yaml
-Uploading 2 files:
-Uploading bundle for /Users/test/redocly/openapi-cli/nodejs/test.yaml...✓ (1/2)
-Uploading /Users/test/redocly/openapi-cli/nodejs/.redocly.lint-ignore.yaml...✓ (2/2)
+#### Pass entrypoint directly
 
-Definition: test.yaml is successfully pushed to Redocly API Registry
+```bash
+openapi push openapi/petstore.yaml @openapi-org/petstore-api@v1
 ```
 
+In this case, `push` will upload to the Redocly API registry only the definition that was passed to the command. The configuration file is ignored.
 
-**Video tutorial: Using the OpenAPI CLI push command**
+#### Pass entrypoint via configuration file
 
+Instead of a full path, you can use an alias assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints. For example, `petstore`:
```

Comment:
```suggestion
Instead of a full path, you can use an alias assigned in the `apiDefinitions` section within your `.redocly.yaml` configuration file as the entrypoint. For example, `petstore`:
```